modules/apps/transfer/keeper/genesis_test.go

```diff
@@ -16,47 +16,47 @@ func (suite *KeeperTestSuite) TestGenesis() {
 	}
 
 	var (
-		denomTraces           types.Traces
+		denoms                types.Denoms
 		escrows               sdk.Coins
-		pathsAndEscrowAmounts = []struct {
-			path   string
+		traceAndEscrowAmounts = []struct {
```

Comment:
feel like this test could be cleaned up, but didn't want to generate too many diffs